feature/amazon/src/main/java/com/revenuecat/purchases/amazon/storeProductConversions.kt

```diff
@@ -78,7 +78,11 @@ internal fun String.parsePriceUsingRegex(): BigDecimal? {
     val matcher = pattern.matcher(this)
     return matcher.takeIf { it.find() }?.let {
         val dirtyPrice = matcher.group()
-        var price = dirtyPrice.replace(" ", "")
+        var price =
+            dirtyPrice.replace(" ", "")
+            .replace(" ", "")
```

Comment:
Looks like one of these is a tab? Maybe document that so it doesn't seem like it's duplicated 😅